catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/forms/security/AbstractPolicyPlugin.java

```diff
@@ -0,0 +1,82 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.catalog.ui.forms.security;
+
+import ddf.catalog.data.Metacard;
+import ddf.catalog.data.Result;
+import ddf.catalog.operation.Query;
+import ddf.catalog.operation.ResourceRequest;
+import ddf.catalog.operation.ResourceResponse;
+import ddf.catalog.plugin.PolicyPlugin;
+import ddf.catalog.plugin.PolicyResponse;
+import ddf.catalog.plugin.StopProcessingException;
+import ddf.catalog.plugin.impl.PolicyResponseImpl;
+import java.io.Serializable;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Provides default behavior for a policy plugin by returning an empty {@link PolicyResponseImpl}
+ * since no policy information needs to be built up.
+ */
+public abstract class AbstractPolicyPlugin implements PolicyPlugin {
```

Comment:
What purpose does this class serve?

Reply:
It makes the system template policy plugin smaller and easier to consume.